docs/gcp-projects.md

```diff
@@ -0,0 +1,57 @@
+# Restore snapshots from GCP across projects
+
+This guide addresses confusion regarding [github issue](https://github.com/vmware-tanzu/velero/issues/1268).
```

Comment:
let's say "this github issue", or "github issue 1268".

Reply:
Imo, this documentation should stand alone, does not need to refer to the issue, especially since this issue contains a lot more in it. I would just remove this line completely.